docs/review-process.md

```diff
@@ -31,14 +31,14 @@ or even just a "I tested this using ... and it works" is very much appreciated.
 Besides, looking at code of others is a great way to learn more about
 Home Assistant.
 
-## Before creating your PR
+## Before creating your code PR
```

Comment:
Maybe `core` sounds better as this only applies to this section

Reply:
Thank you for the catch!